### PR TITLE
Add .gitignore file

### PR DESCRIPTION
**Pull request by AStilmock**
Add .gitignore file and update associated test.rb files

### CODE REVIEW

**Comment by jgthomas-12** [comment]
Thanks for driving during the meeting with Erin today, Andy. Looks like we should be all set to crush this project!

**Review by jgthomas-12 — APPROVED** [review]
Stoked that we got to the .gitignore first! Let's gooooo!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/coverage
```
Comment (Jredish11) [inline .gitignore:1]: good thing we did this first.  all looks good

**Review by Jredish11 — COMMENTED** [review]
all good! gunna merge

**Review by Jredish11 — APPROVED** [review]

**Pull request merged**